pkg/generator/ingress_translator.go

```diff
@@ -255,20 +255,22 @@ func (translator *IngressTranslator) translateIngress(ctx context.Context, ingre
 		}
 
 		var virtualHost, virtualTLSHost *route.VirtualHost
+		// TODO(norbjd): do we want to enable this by default?
```

Comment:
I'm waiting for your opinion on this. Either 1. we enable this by default, and we don't need the option logic; or 2. we make it configurable through Kourier configmap.